diff --git a/components/Sidebar/OrgImage.tsx b/components/Sidebar/OrgImage.tsx
index f371f49..4fdf674 100644
--- a/components/Sidebar/OrgImage.tsx
+++ b/components/Sidebar/OrgImage.tsx
@@ -47,7 +47,7 @@ export const OrgImage = (props: OrgImageProps) => {
 
   return (
     <Container my={4} position="relative">
-      <img alt="Wow, an image." src={`http://localhost:35901/img/${encodedPath}`} />
+      <img alt="Wow, an image." src={srcName} />
     </Container>
   )
 }
